BALSAMIC/utils/qc_check.py

```diff
@@ -0,0 +1,144 @@
+import pandas as pd
+import re
+import numpy as np
+import os
+import click
+
+
+qc_list = ['MEAN_TARGET_COVERAGE', 'FOLD_80_BASE_PENALTY', 'PCT_OFF_BAIT']
+
+
+def read_hs_metrics(hs_metrics_file):
+
+    '''
+    input: string
+    output: DataFrame
+
+    Reads the HS metrics (json-format) and returns it as a DataFrame
+    '''
```

Comment:
Please use the following format for all other functions below.

```suggestion
    ''' Reads the HS metrics (json-format) and returns it as a DataFrame
    
    Args:
        hs_metrics_file: A string path to the file

    Returns:
        metrics_df: DataFrame
    '''
```